org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/logs/LogHelper.java

```diff
@@ -123,7 +123,7 @@ public static FileHandler getFileHandler(String filePath) throws SecurityExcepti
 			FileHandler fh = null;
 			fh = new FileHandler(filePath, true);
 			fh.setFormatter(new SimpleFormatter());
-			fh.setLevel(Level.INFO);
+			fh.setLevel(getLogLevel());
```

Comment:
I didn't know very good FileHandler, but I wonder if we just remove `fh.setLevel(Level.INFO);` it will take the level of root?

Reply:
Good point.   I can try to confirm a bit later.

Reply:
@scottkurz had you time to investigate this idea?

Reply:
No, I haven't yet, sorry.

Reply:
@scottkurz we are planning on doing a 0.25.0 for some bug fixes and API changes. Do you want to get this PR in for the release?

Reply:
Thanks for asking.  I can't ask you to wait since not sure when I'll get back to this.  Appreciate it though.

Reply:
Okay, sound good!